CHANGELOG.md

```diff
@@ -77,6 +79,13 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
   `ProxyTestCase`; use the traits `CacheAssertions` and `HttpCaller` instead.
 * Added HTTP method parameter to `HttpCaller::getResponse()`.
 
+2.0.0-beta3
+-----------
+
+* **BC break:** The `ResponseTagger` now expects an instance of
+  `TagHeaderFormatter` instead of `TagCapable` as first argument.
+* Added `HashGenerator` interface.
```

Comment:
lets not have this line here. (i agree with the beta3 title and the bc break note on ResponseTagger, lets keep that one)